src/main/java/org/jitsi/videobridge/AbstractEndpoint.java

```diff
@@ -74,6 +77,22 @@
      */
     private boolean expired = false;
 
+    /**
+     * The instance responsible for bookkeeping of who (which receiver) is
```

Comment:
When reading the java doc, I would like to find out how this relates to this AbstractEndpoint